src/chapters/configuring.xml

```diff
@@ -5307,6 +5307,50 @@ guaclog: INFO: All files interpreted successfully.</computeroutput>
                         </listitem>
                     </itemizedlist>
                 </section>
+                <section xml:id="ldap-tokens">
+                    <title>LDAP Extension Tokens</title>
+                    <indexterm>
+                        <primary>tokens</primary>
+                        <secondary>ldap</secondary>
+                    </indexterm>
+                    <para>The LDAP extension will read user attributes provided by the LDAP
+                        server and specified in the <filename>guacamole.properties</filename>
+                        file.  The attributes retrieved for a user are configured using the
+                        <param>ldap-user-attributes</param> parameter.  The user must be able
+                        to read the attribute values from their own LDAP object.  The token
+                        name will be prepended with the <constant>LDAP_</constant> prefix.  As
+                        an example, configuring the following line in
+                        <filename>guacamole.properties</filename> file will produce the
+                        below tokens that can be used in connection configurations:</para>
+                    <programlisting>ldap-user-attributes: cn, givenName, sn, mobile, mail</programlisting>
```

Comment:
I think `<programlisting>` normally needs to be within `<informalexample>` when used in this way. I would also recommend splitting things across where they are actually referenced, as it feels a bit odd to read a sentence which notes and is followed by two examples.

Reply:
Okay, added `<informalexample>` and split the paragraph.